src/app.js

```diff
@@ -5,13 +5,33 @@ import cors from 'cors';
 import logger from 'morgan';
 import passport from 'passport';
 import dotenv from 'dotenv';
+import socketIO from 'socket.io';
 import session from 'express-session';
 import './utilities/socialAuthStrategies';
 import routes from './routes';
 
 // Create global app object
 const app = express();
 
+// let's start our server...
+const server = app.listen(process.env.PORT || 4001, () => {
+  console.log(`Server is up and running on port ${server.address().port}...!`); //eslint-disable-line
+});
+
```

Comment:
I feel we should start our server after we have added all necessary middlewares and routes for the app. I feel starting our server should be the last thing/logic in our app.js file. It's just a suggestion.

Reply:
The server variable was needed in downstream operations and the application works as it should. I will do some more consultation but I would leave it as it is for now. It's something to consider in the future.